eng/publish.proj

```diff
@@ -23,8 +30,22 @@
     <PackagesToPublish Include="$(FinalPublishPattern)" Exclude="@(PackagesToPublish)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <SymbolPackagesToPublish  Include="$(SymbolsPublishPattern)">
+      <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
+    </SymbolPackagesToPublish>
+
+    <PublishSymbolsDependsOn Include="SetupPublishSymbols" />
+  </ItemGroup>
+
   <Target Name="PublishToAzureBlobFeed">
     <Error Condition="'@(PackagesToPublish)'==''" Text="PackagesToPublish for packages is empty." />
+
+    <!-- Include Symbols when publishing to blob feed so that the manifest contains symbols and we send all the assets to BAR -->
+    <ItemGroup>
+      <PackagesToPublish Include="@(SymbolPackagesToPublish)" />
+    </ItemGroup>
```

Comment:
Do we want to include the symol packages always, or should there be some condition on whether or not we're publishing to the blob feed?